sympy/solvers/ode/single.py

```diff
@@ -839,11 +839,11 @@ def _get_general_solution(self, *, simplify_flag: bool = True):
         (C1,) = self.ode_problem.get_numbered_constants(num=1)
         if n==1:
             gensol = Eq(log(fx), (
-            C1 + Integral((-P + Q), x)
+            C1 + Integral((-P - Q), x)
```

Comment:
Can this be simplified to `C1 - Integral(P+Q, x)`, or is that not possible here/ going against convention for this particular problem?

Reply:
Yeah I think it is a good suggestion, should work. I'll test it.

Reply:
https://github.com/sympy/sympy/blob/1409fcf748d8d73014cca509f2816acb82af7fc0/sympy/solvers/ode/tests/test_ode.py#L63
Apart from this failing, all other tests seem fine. 

I am not sure if it will not cause a problem though. Because we have just changed the sign of Q from general equation, it just counteracts that change, so we have the same expression as we have had from the all the previous sympy versions.